src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/_help.py

```diff
@@ -88,11 +99,18 @@
     type: group
     short-summary: Commands to manage deployment operations
 """
-helps['resource provider'] = """
+helps['provider'] = """
     type: group
-    short-summary: Commands to manage resource providers
+    short-summary: Manage resource providers
+"""
+helps['provider register'] = """
```

Comment:
I see your point. It was still flagged for correction in the command review with ARM team, mainly to get concept right with the benefit of easier discover thereafter, also node cli does the same